ogc-common/src/main/java/io/arlas/server/exceptions/OGC/OGCException.java

```diff
@@ -88,6 +89,7 @@ public OGCException(OGCExceptionCode exceptionCode, String exceptionText, Servic
     }
 
     public OGCException(List<OGCExceptionMessage> exceptionMessages, Service service) {
+        super(exceptionMessages.stream().filter(em -> (em.getExceptionTexts() != null && !em.getExceptionTexts().isEmpty())).map(em -> em.getExceptionTexts().stream().reduce((a, b) -> a + "\t" + b).orElse("")).reduce((a, b) -> a + "\t" + b).orElse(""));
```

Comment:
Did you test with .collect(Collectors.joining("\t")) ? It may avoid the need of the `orElse` and it is more readable.
There may be also some CollectionUtils.isEmpty(em.getExceptionTexts()) in order to avoid the need of the null check.

Reply:
Fixed